public/contextMenu.js

```diff
@@ -0,0 +1,409 @@
+/*
+MIT License
+
+Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (https://sindresorhus.com)
+Original from: https://github.com/sindresorhus/electron-context-menu
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documen
+    tation files (the "Software"), to deal in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+    persons to whom the Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the
+    Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+    WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS
+    OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+    OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+const electron = require('electron');
```

Comment:
I think we should keep only the minimum required for our feature and not the whole file. Besides that, would be interesting to add support for `debugMode`. We have this in our `public/electron.js` and this could add the 'Inspect element' in the context menu.

Reply:
The features we did not use were now removed from our code.

The overall library architecture was kept despite being a bit over-sized for our needs because it is already a stable code, handles edge cases and has good maintenance.